mem0/memory/main.py

```diff
@@ -735,3 +738,771 @@ def reset(self):
 
     def chat(self, query):
         raise NotImplementedError("Chat function not implemented yet.")
+
+
+class AsyncMemory(MemoryBase):
+    def __init__(self, config: MemoryConfig = MemoryConfig()):
+        self.config = config
+
+        self.custom_fact_extraction_prompt = self.config.custom_fact_extraction_prompt
```

Comment:
Why do you need this? Can directly refer to `self.config.custom_fact_extraction_prompt`